pageserver/src/bin/pageserver.rs

```diff
@@ -495,6 +499,50 @@ fn cli() -> Command {
         )
 }
 
+/// Named symbol for our panic hook, which logs the panic.
+fn tracing_panic_hook(info: &std::panic::PanicInfo) {
+    // following rust 1.66.1 std implementation:
+    // https://github.com/rust-lang/rust/blob/90743e7298aca107ddaa0c202a4d3604e29bfeb6/library/std/src/panicking.rs#L235-L288
+    let location = info.location();
+
+    let msg = match info.payload().downcast_ref::<&'static str>() {
+        Some(s) => *s,
+        None => match info.payload().downcast_ref::<String>() {
+            Some(s) => &s[..],
+            None => "Box<dyn Any>",
+        },
+    };
+
+    let thread = std::thread::current();
+    let thread = thread.name().unwrap_or("<unnamed>");
+    let backtrace = std::backtrace::Backtrace::capture();
```

Comment:
```suggestion
    # Use force_capture: we want the stack trace in the logs, regardless of
    # whether RUST_BACKTRACE is set or not.
    let backtrace = std::backtrace::Backtrace::force_capture();
```

Reply:
We have that setting already set in deployments, what if you are manual testing and want to hit something often? That'd be altering the existing behaviour, so don't like it.

Reply:
Resolving.